app/controllers/assignments_controller.rb

```diff
@@ -83,7 +83,10 @@ def set_unlinked_users
     return unless @organization.roster
 
     assignment_users = @assignment.users
-    roster_entry_users = @organization.roster.roster_entries.map(&:user).compact
+
+    roster_entries = @organization.roster.roster_entries
+    user_ids = roster_entries.map(&:user_id)
+    roster_entry_users = User.where(id: user_ids)
```

Comment:
Is it possible to use `pluck` here instead of mapping through all of the records getting the `user_id`?

```ruby
roster_entry_user_ids = @organization.roster.roster_entries.pluck(:user_id)
roster_entry_users = User.where(id: roster_entry_user_ids)
```